meson.build

```diff
@@ -106,6 +114,11 @@ labwc_deps = [
   math,
   png,
 ]
+if svg.found()
+  labwc_deps += [
+    svg,
+  ]
+endif
```

Comment:
Should this be `if have_rsvg` rather than `svg.found()`? Otherwise it appears we are linking librsvg if its found but disabled by the user.

Reply:
Oh. Good catch indeed. Done.